tests/dogtag/pytest-ansible/provision/osp_vars.py

```diff
@@ -0,0 +1,10 @@
+OS_AUTH_URL = "https://rhos-d.infra.prod.upshift.rdu2.redhat.com:13000/v3"
```

Comment:
Could we get this from environment variable?

Reply:
Thanks @edewata  All the variables are read from ENV variables.